Author: gregor herrmann <gregoa@debian.org>
Description three_argument_open needs 5.6, otherwise the pmv test fails
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=52700
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=560616
Forwarded: http://rt.cpan.org/Public/Bug/Display.html?id=52700
Origin: vendor

--- a/lib/Module/Plan/Base.pm
+++ b/lib/Module/Plan/Base.pm
@@ -22,7 +22,7 @@
 
 =cut
 
-use 5.005;
+use 5.006;
 use strict;
 use Carp           'croak';
 use File::Spec     ();
--- a/META.yml
+++ b/META.yml
@@ -37,7 +37,7 @@
   PAR::Dist: 0.25
   Params::Util: 0.21
   URI: 0
-  perl: 5.00503
+  perl: 5.006
 resources:
   ChangeLog: http://fisheye2.atlassian.com/changelog/cpan/trunk/pip
   license: http://dev.perl.org/licenses/
